packages/cli/src/commands/container/rm.ts

```diff
@@ -1,6 +1,8 @@
 import {Command, flags} from '@heroku-cli/command'
 import {ux} from '@oclif/core'
 import color from '@heroku-cli/color'
+import * as helpers from '../../lib/container/helpers'
```

Comment:
It looks like you did this in all files, could you please update them all to just import the function you need?